marklogic-data-hub-central/ui/e2e/qa-project/gradle-dhs.properties

```diff
@@ -0,0 +1,45 @@
+mlDHFVersion=5.3-SNAPSHOT
+
+mlIsHostLoadBalancer=true
+mlIsProvisionedEnvironment=true
+mlStagingAuth=basic
+mlFinalAuth=basic
+mlJobAuth=basic
+mlAppServicesAuthentication=basic
+mlJobSimpleSsl=true
+mlFinalSimpleSsl=true
+mlStagingSimpleSsl=true
+mlAdminScheme=https
+mlManageScheme=https
+mlAppServicesSimpleSsl=true
+mlManageSimpleSsl=true
+sslFlag=true
+
+mlStagingAppserverName=data-hub-STAGING
+mlStagingPort=8010
+mlStagingDbName=data-hub-STAGING
+mlStagingForestsPerHost=1
+
+mlAppServicesPort=8010
+
+mlJobAppserverName=data-hub-JOBS
+mlJobPort=8013
+mlJobDbName=data-hub-JOBS
+mlJobForestsPerHost=1
+
+mlModulesDbName=data-hub-MODULES
+mlModulePermissions=flowDeveloper,read,flowDeveloper,execute,flowDeveloper,insert,flowDeveloper,update,flowOperator,read,flowOperator,execute,flowOperator,insert,flowOperator,update
```

Comment:
This is incorrect; remove it so that the correct default value is used. Also remove mlModulesDbName.